app/src/main/java/nextstep/shoppingcart/ui/cart/CartItem.kt

```diff
@@ -0,0 +1,135 @@
+import androidx.compose.material3.CardDefaults
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.OutlinedCard
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import coil3.compose.AsyncImage
+import nextstep.shoppingcart.domain.CartProduct
+import nextstep.shoppingcart.ui.component.CountUpdateButton
+import nextstep.shoppingcart.ui.theme.Gray10
+import nextstep.shoppingcart.ui.theme.ShoppingCartTheme
+import nextstep.shoppingcart.ui.theme.Typography
+import nextstep.signup.R
+
+@Composable
+fun CartItem(
+    cartProduct: CartProduct,
+    onCountMinus: () -> Unit,
+    onCountPlus: () -> Unit,
+    onItemDelete: () -> Unit,
+) {
+    OutlinedCard(
+        modifier = Modifier.fillMaxWidth(),
+        border = BorderStroke(1.dp, Gray10),
+        colors = CardDefaults.cardColors().copy(containerColor = Color.White),
+    ) {
+        Column(
+            modifier = Modifier.fillMaxWidth(),
+        ) {
+            Row(
+                modifier =
+                    Modifier
+                        .fillMaxWidth()
+                        .padding(start = 18.dp, end = 6.dp, top = 6.dp),
+                horizontalArrangement = Arrangement.SpaceBetween,
+            ) {
+                Text(
+                    modifier =
+                        Modifier
+                            .align(Alignment.CenterVertically)
+                            .weight(1f),
+                    text = cartProduct.product.name,
+                    style = Typography.titleMedium,
+                    maxLines = 1,
+                    overflow = TextOverflow.Ellipsis,
+                )
+
+                IconButton(
+                    modifier =
+                        Modifier
+                            .height(48.dp)
+                            .width(48.dp),
+                    onClick = onItemDelete,
+                ) {
+                    Icon(
+                        modifier =
+                            Modifier
+                                .height(24.dp)
+                                .width(24.dp),
+                        imageVector = Icons.Filled.Close,
+                        contentDescription = stringResource(R.string.product_list_cart_button_desc),
+                    )
+                }
+            }
+
+            Row(
+                modifier =
+                    Modifier
+                        .padding(horizontal = 18.dp)
+                        .padding(bottom = 18.dp)
+                        .fillMaxWidth(),
```

Comment:
아래와 같이 구현하는 것과 무엇이 다르다고 생각하시나요?
```suggestion
                        .padding(bottom = 18.dp, start = 18.dp, end = 18.dp)
                        .fillMaxWidth(),
```

Reply:
Spacer vs Padding도 고민해봐야 합니다.
https://www.jetpackcompose.app/articles/space-hoisting-should-I-use-the-padding-modifier-or-the-spacer-composable#space-hoisting
> A lot of components react to gestures and animations. Imagine a row component that shows the ripple effect when you tap on it. If we were to pass in the padding Modifier to that component, even this extra space that we added would react to the gesture. As the order of the Modifiers matter, you might be able to avoid this problem, but it would depend on the implementation. Using Spacer simplifies this and avoids the problem altogether.

예를 들어, 탭할 때 리플 효과를 보여주는 Row 컴포넌트에 패딩 Modifier를 적용하면 리플 효과가 예상치 못하게 적용될 수 있습니다. Modifier의 순서를 잘 조절하면 구현에 따라 이 문제를 피할 수 있지만, Spacer를 사용하면 이 문제를 간단히 해결하고 완전히 피할 수 있다고 설명하네요!